config/initializers/job_configurations.rb

```diff
@@ -223,7 +223,7 @@
         cron: cron_24h,
         args: -> { [Time.zone.yesterday.end_of_day] },
       },
-      # Monthly drop of report
+      # Weekly drop of report
       drop_off_report: {
         class: 'Reports::DropOffReport',
         cron: cron_24h,
```

Comment:
Does this mean that the report runs every night to show the past week's worth of data?

Reply:
I'm wondering about this too..